sale_triple_discount/views/sale_group_invoice_discount.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.ui.view" id="invoice_triple_discount_form_view">
+        <field name="name">Account Invoice</field>
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account_invoice_triple_discount.invoice_triple_discount_form_view"/>
+        <field name="arch" type="xml">
+            <data>
+                <xpath expr="//field[@name='discount2']" position="attributes">
+                    <attribute name="groups">sale.group_discount_per_so_line</attribute>
```

Comment:
Doubt: this will override already assigned group [here](https://github.com/OCA/account-invoicing/blob/12.0/account_invoice_triple_discount/views/account_invoice_view.xml#L37)?

Reply:
Also the Odoo original discount group is base.group_no_one but it is changed in sale.group_discount_per_so_line in sale module. 
So I think base.group_no_one is added to respect the original Odoo behavior but it is needs to be changed in sale.group_discount_per_so_line like sale module does for original diacount field.
And for the same behaviour the changed is done in sale and so sale_triple_discount.

Reply:
@sergiocorato don't you agree with me?

Reply:
[Here](https://github.com/OCA/account-invoicing/pull/915) you can find the reason of this PR.

Reply:
I think it's ok, thanks for the info.

Reply:
> [Here](https://github.com/OCA/account-invoicing/pull/915) you can find the reason of this PR.

Could you review this one?